src/entities/Entity.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * Abstract class to create an entity with unique properties
```

Comment:
Represents an object or resource in the Polymath Ecosystem with its own set of properties and functionality